.github/workflows/builds.yaml

```diff
@@ -0,0 +1,189 @@
+name: Builds
+
+on: push
+
+jobs:
+  debian:
+    name: Debian & Ubuntu
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.distro }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        pd_multi: [ON, OFF]
+        build_type: [Debug, Release]
+        distro: ["debian:bookworm", "ubuntu:lunar", "ubuntu:mantic"]
```

Comment:
For Debian, you should use `stable` and `testing` rather than hardcoding a release name (and I think you should test both).
Unfortunately this is not possible for Ubuntu (though I'm sure that I have some code lying around somewhere to create a build matrix with a dynamically calculated list of supported/current releases)

Reply:
done

Reply:
(although personally I've been bitten enough by platforms which change under my feet to prefer fixed names - e.g. now the build status for a given commit could change in 5 years when debian:stable refers to a new distro)

Reply:
which it should. so everything is good now. thanks